source/compatibility.tex

```diff
@@ -2678,7 +2677,7 @@
 The types \tcode{char16_t} and \tcode{char32_t}
 are distinct types rather than typedefs to existing integral types.
 The tokens \tcode{char16_t} and \tcode{char32_t}
-are keywords in this International Standard\iref{lex.key}.
+are keywords in \Cpp\iref{lex.key}.
```

Comment:
You used braces after `\Cpp` in the other edits. I understand it probably doesn't make a difference here, but for dumb user consistency, it would be nice to have the braces everywhere.